Roles/Impostor/Butcher.cs

```diff
@@ -55,11 +58,12 @@ public override void OnMurderPlayerAsKiller(PlayerControl killer, PlayerControl
             var pcList = Main.AllAlivePlayerControls.Where(x => x.PlayerId != target.PlayerId); //No need to do extra check cause nobody is winning
             pcList.Do(x =>
             {
+                Main.PlayerStates[x.PlayerId].IsDead = true;
```

Comment:
Better use `.SetDead();` becouse modded clients not synced

Reply:
I just dont want to send rpc here lol

Reply:
Hmm